modules/sdk/src/main/scala/org/tessellation/sdk/domain/snapshot/services/GlobalL0Service.scala

```diff
@@ -23,48 +37,128 @@ import eu.timepit.refined.types.numeric.{NonNegLong, PosLong}
 import org.typelevel.log4cats.slf4j.Slf4jLogger
 
 trait GlobalL0Service[F[_]] {
-  def pullLatestSnapshot: F[(Hashed[GlobalIncrementalSnapshot], GlobalSnapshotInfo)]
-  def pullGlobalSnapshots: F[Either[(Hashed[GlobalIncrementalSnapshot], GlobalSnapshotInfo), List[Hashed[GlobalIncrementalSnapshot]]]]
+  type LatestSnapshotTuple = (Hashed[GlobalIncrementalSnapshot], GlobalSnapshotInfo)
+  def pullLatestSnapshot: F[LatestSnapshotTuple]
+  def pullGlobalSnapshots: F[Either[LatestSnapshotTuple, List[Hashed[GlobalIncrementalSnapshot]]]]
   def pullGlobalSnapshot(ordinal: SnapshotOrdinal): F[Option[Hashed[GlobalIncrementalSnapshot]]]
   def pullGlobalSnapshot(hash: Hash): F[Option[Hashed[GlobalIncrementalSnapshot]]]
 }
 
 object GlobalL0Service {
+  val minimumMajorityPeerIds = 2
```

Comment:
I think it should also work with a single peer. At some point users may want to have their own nodes on the GL0 as a alignment point.

Reply:
For a single peer and two peers cases, all need to agree and declare the same hash simply.

Reply:
the minimum requirement to run a network is a single peer, so it should somehow work for a single peer as well (well, 100% majority in that case)